packages/docusaurus-theme-classic/src/index.js

```diff
@@ -22,6 +22,16 @@ class DocusaurusThemeDefault {
   getThemePath() {
     return path.resolve(__dirname, './theme');
   }
+
+  configureWebpack() {
```

Comment:
This is how I provide a fallback alias in case `theme-classic` is used but `theme-algolia-search` is not. Doing this accidentally made this alias a fallback as this is part of the normal plugin lifecycle whereas aliases created in `getThemePath()` only come later on and the `getThemePath()` from `theme-algolia-search` will overwrite the one here. I didn't expect this but it was a pleasant surprise. It doesn't mean I want to keep things the way they are now though. 😂

Reply:
What if the implementation isn't done through this configureWebpack ?

Example:
I created a theme-endilie that a default search bar with alias @theme/Search
Then I add another theme-search-algolia that has the alias @theme/Search as well

Can the theme-search-algolia override the theme-endilie search ? Assume i will put it in correct order and there is no configureWebpack alias like this PR in theme-endilie